tools/cicd/cloudbuild.yaml

```diff
@@ -25,18 +26,21 @@ steps:
   - --build-arg
   - PYTHON_VERSION=${_PYTHON_VERSION}
   - .
+  waitFor: ['setup-build']
 
 - name: gcr.io/$PROJECT_ID/hatch:${_PYTHON_VERSION}
   id: tests
```

Comment:
The `tests` and `lint` steps requires the hatch images build by the step at line #16.